src/utils/sortIdentities.ts

```diff
@@ -7,6 +7,10 @@ function sortPrimaryIdentity(a: Identity, b: Identity): number {
   return a.primary === b.primary ? 0 : a.primary ? -1 : 1;
 }
 
+function sortStatus(a: Identity, b: Identity): number {
```

Comment:
Could you rename this as `sortIdentityStatus`?

Reply:
OK fixed 1978b8d